### DISCUSSION TITLE
Just a warm applause for this project 🙌

### DISCUSSION THREAD

**Discussion by luissalgadofreire** [jsakamoto/BlazingStory#45] [General]
Hi.

Just a thank you for this project and hopes that it will grow and thrive. I started using it a few days ago and it is already very useful to me. Best of wishes for the project.

**Comment by jsakamoto** [comment]
Hi @luissalgadofreire, 
Thank you for your kind words!

Since the "Blazing Story" is my private hobby product, my time and resources for it are significantly limited. I also released many other OSS products and need to maintain them for a long time, as well as the "Blazing Story". Moreover, I have different ideas for Blazor applications and libraries that I want to implement. In short, I have a lot of things that I want to do, and time flies!

Therefore, the growth speed of the "Blazing Story" must be much slower unless other developers with rich resources fork over this product. In fact, the current version of the "Blazing Story" is still buggy and has many limitations.

However, I'll try to keep improving the "Blazing Story" even though its growing speed is not so fast.

Thanks everyone for your patience.
